eng/common/scripts/git-branch-push.ps1

```diff
@@ -27,7 +27,8 @@ param(
     [Parameter(Mandatory = $false)]
     [string] $PushArgs = "",
 
-    [string] $RemoteName = "azure-sdk-fork",
+    # [Parameter(Mandatory = $false)]
+    # [string] $RemoteName = "azure-sdk-fork",
```

Comment:
This might be a breaking change. Can you verify that we aren't passing this anywhere?

Reply:
I didn't see any usages of it when it was added https://github.com/Azure/azure-sdk-tools/commit/60121995ebeffb4ebb6115a745eb3cb357cae9c4 and from a quick search https://github.com/Azure/azure-sdk-tools/search?q=RemoteName. So it is probably OK make this change.